tests/react/vanilla-utils/atomWithStorage.test.tsx

```diff
@@ -445,6 +445,62 @@ describe('atomWithStorage (with browser storage)', () => {
 
     expect(store.get(dummyAtom)).toBe(1)
   })
+
+  it("should recompute dependents' state after onMount (#2098)", async () => {
+    const store = createStore()
+    const mockNativeStorage = Object.create(window.Storage.prototype)
+    let currentValue: string | null = 'true'
+    mockNativeStorage.setItem = vi.fn(
+      (_key: string, value: string) => (currentValue = value)
+    )
+    mockNativeStorage.getItem = vi.fn(() => currentValue)
+    mockNativeStorage.removeItem = vi.fn(() => (currentValue = null))
+
+    const isLoggedAtom = atom(false)
+    const _isDevModeState = atomWithStorage(
+      '_isDevModeState',
```

Comment:
We reserve `_foo` for unused names:
```suggestion
    const isDevModeState = atomWithStorage(
      'isDevModeState',
```